mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -6575,6 +6598,9 @@ func flattenNodePoolAutoConfig(c *container.NodePoolAutoConfig) []map[string]int
 	}
 
 	result := make(map[string]interface{})
+	if c.NodeKubeletConfig != nil {
+		result["node_kubelet_config"] = flattenKubeletConfig(c.NodeKubeletConfig)
```

Comment:
this needs to be updated to a separate function, or we get a kernel panic here... will push up a fix shortly.

Reply:
With that update:
```
--- PASS: TestAccContainerCluster_withAutopilotKubeletConfig (1921.53s)
PASS
ok  	github.com/hashicorp/terraform-provider-google/google/services/container	1922.494s
```